### ISSUE TITLE
Multiple instances : nodes can't communicate

### ISSUE THREAD

**Issue by rmjstn** [RocketChat/Rocket.Chat#13301]
### Description:

We are running a multi-instances of Rocket.Chat (new installation). Two Docker containers.
INSTANCE_IP and extra_hosts are set in the Docker Compose file.

### Steps to reproduce:

Notifications and users status doesn’t work properly. In the “Broadcast Connected Instances” section “Auth” and “Current Status > Connected” are not set, and "“Current Status > Status” is set to waiting.

I found this in the logs : 

First Docker container : 
```
stream error Can't establish a connection to the server at ws://10.241.1.9:3000/websocket
```

Second one : 
```
stream error Can't establish a connection to the server at ws://10.241.1.7:3000/websocket
```

### Actual behavior:

Informations on start up :
```
Rocket.Chat Version: 0.74.0                       
NodeJS Version: 8.11.4 - x64                 
Platform: linux                        
Process Port: 3000                         
Site URL: https://rocket-test.xxx.xx/  
ReplicaSet OpLog: Enabled                      
Commit Hash: cc5391e58c                   
Commit Branch: HEAD       
```                  
 
From the first container, I can join the second one on port 3000 :
```
rocketchat@865f8ce4ac9b:/app/bundle$ > /dev/tcp/10.241.1.9/3000
```
I see network packets on the second server.

I've installed [wscat](https://github.com/websockets/wscat) in the container. It works, I can open the websocket :

```
> rocketchat@865f8ce4ac9b:/app/bundle$ wscat -c ws://10.241.1.9:3000/websocket
connected (press CTRL+C to quit)
< {"server_id":"0"}
```

When I start the container on the first server, I don't see any network packets on port 3000 on the second one.

One of the Docker Compose file : 
```
rocketchat:
  image: rocketchat/rocket.chat:latest
  environment:
    - MONGO_URL=mongodb://rocketchat:toto@mongo345:27017/rocketchat?authSource=admin&replicaSet=rs01&readPreference=nearest&w=majority
    - MONGO_OPLOG_URL=mongodb://oplog:toto@mongo345:27017/local?authSource=admin&replicaSet=rs01
    - ROOT_URL=https://rocket-test.xxxxx.xx
    - PORT=3000
    - INSTANCE_IP=10.241.1.7
  ports:
    - 3000:3000
  extra_hosts:
    - "rocket-1:10.241.1.7"    
    - "rocket-2:10.241.1.9"    
```

### Server Setup Information:

- Version of Rocket.Chat Server: 0.74.0
- Operating System: Debian GNU/Linux 9
- Deployment Method: Docker
- Number of Running Instances: 2
- DB Replicaset Oplog: enabled
- NodeJS Version: 8.11.4 - x64 
- MongoDB Version: 3.4.5

### Additional context

In summary : 

1.  From container1, check open port : packets capture on server 2.
2.  From container 1, call websocket on container 2 (via wscat) : packets capture on server 2.
3. Start container 1, error on websocket connection : NO packets capture on server 2.

### Relevant logs:

```
stream error Can't establish a connection to the server at ws://10.241.1.9:3000/websocket
```

Thanks,

Rémy

**Comment by rmjstn** [comment]
### Additional information

I made a packet capture in the container (eth0@if101 interface). When we had the "Can't establish a connection to the server" error, we don't see any traffic.

Rémy

**Comment by geekgonecrazy** [comment]
From the container can you hop in it and actually hit that port?  like curl http://rocket-1:3000/api/info ?

This seems like an issue with the network layer

**Comment by rendxq** [comment]
I have the same problem
Two Rocket.Chat instances (rocketchat, rocketchat1).
From the containers I can get data:
 "curl http://rocketchat:3000/api/info ---> {"version":"1.3.2","success":true}"
 "curl http://rocketchat1:3000/api/info ---> {"version":"1.3.2","success":true}"

Need help. It there a solution to troubleshoot it?

**Comment by rmjstn** [comment]
> From the container can you hop in it and actually hit that port? like curl http://rocket-1:3000/api/info ?
> 
> This seems like an issue with the network layer

Hi,

Sorry for the delay.

I made an update. I've got the same problem.
Here is what I get : 
curl http://10.241.1.7:3000/api/info : {"version":"1.3.2","success":true}
curl http://10.241.1.9:3000/api/info : {"version":"1.3.2","success":true}

Rémy

**Comment by rendxq** [comment]
After update to v2.3.2 the issuers still are existing.

"curl http://rocketchat:3000/api/info ---> {"version":"1.3.2","success":true}"
"curl http://rocketchat1:3000/api/info ---> {"version":"1.3.2","success":true}"
but in rocketchat log I see:
stream error Can't establish a connection to the server at ws://rocketchat:3000/websocket
stream error Can't establish a connection to the server at ws://rocketchat1:3001/websocket


Current Status  -> waiting

Has somebody working solution?

**Comment by geekgonecrazy** [comment]
this seems to be something specific to configuration since more more reports.  Please use one of the support methods in the docs to work through this: https://rocket.chat/docs/getting-support/#community-support

**Comment by 564064202** [comment]
please give us your total docker-compose.yml
